packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,324 @@
+# Checkout
+Checkout is a process of providing shipping and billing addresses and selecting shipping and payment methods needed to place an order and pay for it.
```

Comment:
**suggestion**: let's have a consistent newline between headings and text and between text and code snippets
```suggestion
# Checkout

Checkout is a process of providing shipping and billing addresses and selecting shipping and payment methods needed to place an order and pay for it.
```